sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -55,32 +59,53 @@ export class AssessmentAESTController extends BaseController {
    */
   @Get("application/:applicationId/requests")
   async getRequestedAssessmentSummary(
-    @Param("applicationId") applicationId: number,
+    @Param("applicationId", ParseIntPipe) applicationId: number,
   ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
+    let requestAssessmentSummary: RequestAssessmentSummaryAPIOutDTO;
+    const offeringChange =
+      await this.educationProgramOfferingService.getOfferingRequestsByApplicationId(
+        applicationId,
+      );
+    if (offeringChange) {
+      requestAssessmentSummary = {
+        id: offeringChange.id,
+        submittedDate: offeringChange.submittedDate,
+        status: offeringChange.offeringStatus,
+        requestType: RequestAssessmentTypeAPIOutDTO.OfferingRequest,
+        programId: offeringChange.educationProgram.id,
+      };
+    }
```

Comment:
IMO the method can be simplified like this with reduced null checks. 

` async getRequestedAssessmentSummary(
    @Param("applicationId", ParseIntPipe) applicationId: number,
  ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
    let requestAssessmentSummary: RequestAssessmentSummaryAPIOutDTO[] = [];
    const offeringChange =
      await this.educationProgramOfferingService.getOfferingRequestsByApplicationId(
        applicationId,
      );
    if (offeringChange) {
      requestAssessmentSummary.push({
        id: offeringChange.id,
        submittedDate: offeringChange.submittedDate,
        status: offeringChange.offeringStatus,
        requestType: RequestAssessmentTypeAPIOutDTO.OfferingRequest,
        programId: offeringChange.educationProgram.id,
      });
    }

    const applicationExceptions =
      await this.applicationExceptionService.getExceptionsByApplicationId(
        applicationId,
        ApplicationExceptionStatus.Pending,
        ApplicationExceptionStatus.Declined,
      );

    if (applicationExceptions.length > 0) {
      const applicationExceptionArray: RequestAssessmentSummaryAPIOutDTO[] =
        applicationExceptions.map((applicationException) => ({
          id: applicationException.id,
          submittedDate: applicationException.createdAt,
          status: applicationException.exceptionStatus,
          requestType: RequestAssessmentTypeAPIOutDTO.StudentException,
        }));
      return requestAssessmentSummary.concat(applicationExceptionArray);
    }

    const studentAppeal =
      await this.studentAppealService.getPendingAndDeniedAppeals(applicationId);
    const studentAppealArray: RequestAssessmentSummaryAPIOutDTO[] =
      studentAppeal.map((appeals) => ({
        id: appeals.id,
        submittedDate: appeals.submittedDate,
        status: appeals.status,
        requestType: RequestAssessmentTypeAPIOutDTO.StudentAppeal,
      }));
    return requestAssessmentSummary.concat(studentAppealArray);
  }`